packages/itinerary-body/src/styled.tsx

```diff
@@ -628,6 +628,12 @@ export const TransitAlert = styled.a`
   margin-top: 5px;
   padding: 8px;
   text-decoration: none;
+  @media only screen and (max-width: 375px) {
+    width: 55%;
+  }
+  @media only screen and (max-width: 480px) {
+    width: 60%;
```

Comment:
How did you come up with these numbers?

Reply:
All these changes were through inspect element on Chrome

Reply:
Btw this CSS does not cover regular (desktop) screen sizes.

Reply:
Good catch yeah. For this PR I'm focusing on the mobile regression since it looked okay on desktop, but I'll consider adding desktop styling if I observe more regressions